debian/precise/foreman/foreman-debug.dirs

```diff
@@ -0,0 +1 @@
+usr/share/foreman/script/foreman-debug.d/
```

Comment:
In the RPMs, foreman-proxy depends on foreman-debug so the package would exist.  Even if it's a recommends and foreman-debug's not installed, will the proxy/plugin package install the file OK?  (It'd just be unused.)